style_guides/js_style_guide.md

```diff
@@ -344,92 +476,69 @@ if (a == '') {
 
 ## Only use ternary operators for small, simple code
 
-And **never** use multiple ternaries together
-
-*Right:*
+And *never* use multiple ternaries together
 
 ```js
-var foo = (a === b) ? 1 : 2;
-```
-
-*Wrong:*
+// good
+const foo = (a === b) ? 1 : 2;
```

Comment:
This example needs to be fleshed out to act as a true alternative to the "bad" example:

```javascript
let foo;

switch (a) {
  case b:
    foo = 1;
    break;
  case c:
    foo = 2;
    break;
  default:
    foo = 3;
}
```